src/renderer/features/contacts/ContactFilter/model/__tests__/contact-filter.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { fork } from 'effector';
+
+import { filterModel } from '@features/contacts';
+import { contactModel } from '@entities/contact';
+
+const mockContact1 = {
```

Comment:
I'd rather stick with a single variable
```javascript
const contacts = [ ... ];
```

Reply:
I did this so that I can use specific contact details in the query. Look at the last 2 tests, they are IMHO much cleaner when I do `getScopeByQuery(mockContact1.address);` rather than `getScopeByQuery(222)` because what we then expect is `mockContact1` rather than some value that aren't clear.

Reply:
``` javascript
const contacts = [ ... ];
// in tests
allSettled(filterModel.events.queryChanged, { scope, params: contacts[0].matrixId });
```